codegenerator/cli/src/hbs_templating/codegen_templates.rs

```diff
@@ -336,22 +336,12 @@ impl EntityRecordTypeTemplate {
 
 #[derive(Serialize, Debug, PartialEq, Clone)]
 pub struct EventTemplate {
-    pub name: CapitalizedOptions,
+    pub name: String,
+    pub module_code: String,
```

Comment:
I moved the codegen to the rust side to have more flexibility on splitting Fuel and Evm code.